examples/upsert-resource/simple/README.md

```diff
@@ -0,0 +1,77 @@
+# upsert-resource: Simple Example
```

Comment:
@mengqiy There's back-and-forth on naming conventions. There's a possibility this will be renamed to 'upsert'. Should hold off on releasing the image until that's settled.